src/corehost/cli/deps_format.cpp

```diff
@@ -134,16 +134,18 @@ bool deps_json_t::perform_rid_fallback(rid_specific_assets_t* portable_assets, c
         {
             if (rid_fallback_graph.count(host_rid) == 0)
             {
-                trace::error(_X("Did not find fallback rids for package %s for the host rid %s"), package.first.c_str(), host_rid.c_str());
-                return false;
+                trace::warning(_X("Did not find fallback rids for package %s for the host rid %s"), package.first.c_str(), host_rid.c_str());
```

Comment:
"rid" is a pretty internal term. How about `The framework 'Microsoft.NETCore.App 1.0.0-abc123' does not support the runtime 'win7-x64'. Some native libraries may fail to load on this platform.`

(where `Microsoft.NETCore.App 1.0.0-abc123` is the shared framework name and version, and `win7-x64` is the host RID)